src/org/centos/pipeline/PackagePipelineUtils.groovy

```diff
@@ -279,6 +279,7 @@ def setStageEnvVars(String stage){
                     fed_branch                : env.fed_branch,
                     fed_repo                  : env.fed_repo,
                     fed_rev                   : env.fed_rev,
+                    fed_id                    : (env.fed_pr_id) ?: '',
```

Comment:
this makes me think we should call it fed_id everywhere and never fed_pr_id

Reply:
I think  fed_pr_id is more clear :) And we have all green now...